Formula/netclient.rb

```diff
@@ -0,0 +1,34 @@
+class Netclient < Formula
+  desc "Automate fast, secure, and distributed virtual networks using WireGuard"
+  homepage "https://netmaker.io"
+  url "https://github.com/gravitl/netmaker/archive/v0.15.0.tar.gz"
+  sha256 "cd6f595365dd409400a61b6e7220133d0ea281db1ecc38ac541bc929f748d53e"
+  license "SSPL-1.0"
+  head "https://github.com/gravitl/netmaker.git", branch: "master"
+
+  depends_on "go" => :build
+  depends_on "wireguard-tools"
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-X 'main.version=v#{version}'"), "netclient/main.go"
+  end
+
+  def caveats
+    <<~EOS
+      Netclient requires a daemon to function properly.
+      This daemon needs to run as root, and can be enabled using:
+        sudo brew services start netclient
+    EOS
+  end
+
+  service do
```

Comment:
This will make sure `sudo` is used
```suggestion
  plist_options startup: true
  service do
```

Reply:
Yes, there's very little difference in service handling on linux

Reply:
It's just confusing naming this like "plist something", as plist does not exist on Linux 😄 .

Reply:
Yeah, that's the last reference to plists that I still need to remove from the service handling.